lib/canonize.js

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {crypto} from './platform.js';
+import {encodeBase64Url} from './helpers.js';
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+
+// input: bytes from PDF417. returns base64Url encoded hash of canonized bytes.
+export async function base64UrlFromPdfBytes({pdfBytes} = {}) {
+  const hashedBytes = await hashFromPdfBytes({pdfBytes});
+  const base64Bytes = encodeBase64Url(hashedBytes);
+  return base64Bytes;
+}
+
+// input: bytes from Pdf. Returns CANONIZED BYTES
+// for privacy reasons, use hashCanonizedBytesFromPdfBytes
+// or base64UrlFromPdf417Bytes if sending data anywhere.
+export function canonizedBytesFromPdfBytes({pdfBytes} = {}) {
```

Comment:
If wonder if we can use shorter names here without any issue?

```suggestion
export function canonicalize({pdfBytes} = {}) {
```